indigo_app/templates/indigo_api/_work_form_content.html

```diff
@@ -542,11 +564,10 @@ <h5 class="card-header">{% trans 'Country and locality (advanced)' %}</h5>
     (function () {
       function refresh(e) {
         htmx.trigger("#work-list-item-{{ work.pk }}", "hx_refresh_work_list_item");
+          document.removeEventListener('hide.bs.offcanvas', refresh);
       }
 
-      document.addEventListener('hide.bs.offcanvas', () => {
-        document.removeEventListener('hide.bs.offcanvas', refresh);
-      });
+      document.addEventListener('hide.bs.offcanvas', refresh);
```

Comment:
@longhotsummer I noticed that the work listing refresh was not working as expected. Was it working for you?

Reply:
possibly not, but i don't recall.